lib/cancan/controller_additions.rb

```diff
@@ -12,7 +12,7 @@ module ClassMethods
       #   end
       #
       def load_and_authorize_resource(*args)
-        cancan_resource_class.add_before_filter(self, :load_and_authorize_resource, *args)
+        cancan_resource_class.add_before_action(self, :load_and_authorize_resource, *args)
```

Comment:
Apologies for delay in responding @Senjai , I thought I had already done so. Yeah good spot there and yes it is worrying that it passes tests. Do you have any suggestions on how I might use *_filter in Rails 3.2 and *_action in Rails 4+. Is it just case of having loads of conditionals throughout the code or is there a better way? I don't mind doing the work.